### DISCUSSION TITLE
GitHub Advanced Security has revoked access for the personal access token

### DISCUSSION THREAD

**Discussion by hoonsor** [community/community#45688] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, I created a public repository for the purpose of Obsidian note image storage. I used a Obsidian community plugin called "Emo" and it upload images through my personal access token. However, after I created my personal access token and used it upload images, the token will be revoked by GitHub Advanced Security immediately. How can I solve the problem? Thanks.

<img width="672" alt="Question" src="https://user-images.githubusercontent.com/56955835/215307498-6eef184c-f704-4a12-81c8-08b9383dcdda.png">

**Comment by jge162** [comment, marked as answer]
Hello @hoonsor sorry to hear your situation. You should reach out to support. Here is a link to [GitHub Token support](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/token-expiration-and-revocation) at the bottom of the page if you don’t see your issue. There is a contact us button. Click on it and submit your request. Bests.

**Reply by hoonsor** [reply]
I'll try, thank you!
